examples/stream-to-file/Cargo.toml

```diff
@@ -11,3 +11,5 @@ tokio = { version = "1.0", features = ["full"] }
 tokio-util = { version = "0.7", features = ["io"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+async-stream = "0.3"
+axum-extra = { path = "../../axum-extra", features = ["fileStream"] }
```

Comment:
Please add a line ending.

Reply:
I reformatted the code and pushed it, I'm rust beginner not very familiar with these ci checks yet, sorry!